plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -219,6 +221,21 @@ public BigQueryConfig setCaseInsensitiveNameMatchingCacheTtl(Duration caseInsens
         return this;
     }
 
+    @NotNull
+    @MinDuration("0m")
+    public Duration getDestinationTableCacheExpiry()
+    {
+        return destinationTableCacheExpiry;
+    }
+
+    @Config("bigquery.destination-table-cache-expiry")
+    @ConfigDescription("Duration for which query and destination table will be cached")
```

Comment:
Duration for which the results of querying a view will be cached

Reply:
Done.